package.json

```diff
@@ -40,6 +40,7 @@
     "passport-jwt": "^2.2.1",
     "passport-local": "^1.0.0",
     "pug": "^2.0.0-beta6",
+    "rand-token": "^0.3.0",
```

Comment:
Yes, we used it in the UserSchema to generate a random API key for users when a new one is created.